scripts/vm_run_tests.psm1

```diff
@@ -636,84 +380,89 @@ function Invoke-ConnectRedirectTestsOnVM
             -StandardUserPassword $StandardUserPassword `
             -UserType $UserType `
             -WorkingDirectory $WorkingDirectory
-        Write-Log "Invoke-ConnectRedirectTest finished on $VM"
-
-    } -ArgumentList (
-        $VMName,
-        $VM1V4Address,
-        $VM1V6Address,
-        $VM2V4Address,
-        $VM2V6Address,
-        $VipV4Address,
-        $VipV6Address,
-        $DestinationPort,
-        $ProxyPort,
-        $StandardUser,
-        $InsecurePassword,
-        $UserType,
-        "eBPF",
-        $LogFileName,
-        $TestHangTimeout,
-        $UserModeDumpFolder) -ErrorAction Stop
-    Stop-ProcessOnVM -VM $VMName -ProgramName $ProgramName
-
-    # Remove standard user on VM1.
-    Remove-StandardUserOnVM -VM $VMName -UserName $StandardUser
-}
+        Write-Log "Invoke-ConnectRedirectTest finished"
+    }
+    $argList = @($VM1V4Address, $VM1V6Address, $VM2V4Address, $VM2V6Address, $VipV4Address, $VipV6Address, $DestinationPort, $ProxyPort, $script:StandardUser, $InsecurePassword, $UserType, $script:WorkingDirectory, $LogFileName, $script:TestHangTimeout, $script:UserModeDumpFolder)
+    Invoke-OnHostOrVM -ScriptBlock $scriptBlock -ArgumentList $argList
 
-function Stop-eBPFComponentsOnVM
-{
-    param([Parameter(Mandatory = $true)] [string] $VMName,
-          [Parameter(Mandatory = $false)] [bool] $VerboseLogs = $false)
-    # Stop the components, so that Driver Verifier can catch memory leaks etc.
-    Write-Log "Stopping eBPF components on $VMName"
+    Stop-ProcessHelper -ProgramName $ProgramName
 
-    $TestCredential = New-Credential -Username $Admin -AdminPassword $AdminPassword
+    # TODO - try with this part removed - we never add the standard user on the VM anyway...
+    # if ($script:ExecuteOnVM) {
+    #     # Remove standard user on VM1.
+    #     Remove-StandardUser -UserName $script:StandardUser
+    # }
```

Comment:
What's this?  Avoid commented out code unless it specifically references a github issue number to clean it up once some feature is done.

Reply:
Thanks for pointing this out - this was intended for cleanup before check-in. I have removed this.